test/unit/pr_checker.test.js

```diff
@@ -381,9 +382,38 @@ describe('PRChecker', () => {
 
       const expectedLogs = {
         warn: [
-          [ 'Changes were pushed since the last review:' ],
+          [ '2 commits were pushed since the last review:' ],
           [ '- src: add requested feature' ],
-          [ '- nit: fix errors' ]
+          [ '- nit: edit mistakes' ]
+        ],
+        info: [],
+        trace: [],
+        error: []
+      };
+
+      const checker = new PRChecker(logger, {
+        pr: firstTimerPR,
+        reviewers: allGreenReviewers,
+        comments: commentsWithLGTM,
+        collaborators,
+        reviews,
+        commits
+      });
+
+      let status = checker.checkCommitsAfterReview();
+      assert.deepStrictEqual(status, false);
+      assert.deepStrictEqual(logger.logs, expectedLogs);
+    });
+
+    it('should only warn last three commits if more than 3 commits', () => {
+      const { commits, reviews } = moreThanThreeCommitsAfterReview;
+      const expectedLogs = {
+        warn: [
+          [ '4 commits were pushed since the last review:' ],
+          [ 'last 3 commits...' ],
```

Comment:
Maybe merge this line with the one above? `4 commits were pushed since the last review, the last 3 commits are:`